src/Polymesh.ts

```diff
@@ -347,7 +347,6 @@ export class Polymesh {
     this.context.setPair(signerToString(signer));
   }
 
-  // TODO @monitz87: remove when the dApp team no longer needs it
```

Comment:
Seems like this is part of the API now, this comment as been in for a while now.